node/sc/bridge_manager.go

```diff
@@ -455,7 +487,8 @@ type BridgeManager struct {
 	receivedEvents map[common.Address][]event.Subscription
 	withdrawEvents map[common.Address]event.Subscription
 	bridges        map[common.Address]*BridgeInfo
-	mu             sync.RWMutex
+	bridgeInfoMu   sync.RWMutex
+	tokenEventMu   sync.RWMutex
```

Comment:
Should we divide a mutex into two? Isn't there any chance of causing a race condition?

Reply:
I'm also confused by name. Is the tokenEventMu mutex's target variable more than one?
The bridgeInfoMu is for bridges variable? How can change bridgesMu? 
I don't know the tokenEventMu's target. Can you explain with comment?

Reply:
@hqjang-pepper, The `tokenEventMu` is for a guard of `receivedEvent` and `withdrawEvents` variable. Can you elaborate on this ` Isn't there any chance of causing a race condition?`? Do you mean a deadlock by lock-release ordering?  
@henry-will,The suggested name seems better. Applied.

Reply:
I've checked all logic `bridgeInfoMu` and `tokenEventMu`, couldn't find possible deadlock point. So I'll resolve this conversation.